<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1539</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="Mainframe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1521</width>
      <height>861</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>83</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black;
background-color: rgb(223, 223, 223);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Démarrer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pauseButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>83</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0.5px solid black;
background-color: rgb(223, 223, 223);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEditer">
     <property name="geometry">
      <rect>
       <x>1390</x>
       <y>10</y>
       <width>118</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QGraphicsView" name="CuisineGraphicsView">
     <property name="geometry">
      <rect>
       <x>885</x>
       <y>50</y>
       <width>631</width>
       <height>741</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black;
background-image: url(:/build/Desktop_Qt_6_8_0_MinGW_64_bit-Debug/debug/images/carreau.jpg);
border-radius: 7px;</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="RestaurantGraphicsView">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>51</y>
       <width>871</width>
       <height>741</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black;
background-image: url(:/build/Desktop_Qt_6_8_0_MinGW_64_bit-Debug/debug/images/parquet3.jpg);
border-radius: 7px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>75</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0.5px solid black;
background-color: rgb(223, 223, 223);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Arrêter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dashboardButton">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>0</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black;
</string>
     </property>
     <property name="text">
      <string>Ouvrir le dashboard</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1539</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="parquet.qrc"/>
 </resources>
 <connections/>
</ui>
